magit-todos.el

```diff
@@ -363,6 +363,12 @@ from the \"topic2\" branch, this option could be set to
   "Show submodule to-do list."
   :type 'boolean)
 
+(defcustom magit-todos-trim-filename-fn nil
```

Comment:
We might as well avoid the abbreviation and name this `magit-todos-trim-filename-function`.

Even better, though, since it's really just a filter and not only for trimming, how about `magit-todos-filename-filter`?